src/styles/blocks/_cards.scss

```diff
@@ -0,0 +1,135 @@
+.container {
```

Comment:
checklist 8th item, 
[BEM] - create a separate file per each BEM block styles that have the same name as the block